src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -308,10 +298,7 @@ public function getOperation($name, $optionalArgs = [])
             $this->descriptors['getOperation']
         );
 
-        return $callable(
-            $request,
-            [],
-            ['call_credentials_callback' => $this->createCredentialsCallback()]);
+        return $callable($request)->wait();
```

Comment:
I think the $callable will need a second arg:
`return $callable($request, [])->wait();`